spec/forms/openid_connect_token_form_spec.rb

```diff
@@ -89,7 +89,7 @@
       end
 
       context 'the code has a null byte' do
-        let(:code) { "\x00code"}
+        let(:code) { "\x00code" }
```

Comment:
extremely surprised we didn't have this already! guess it was just the spaces inside hashes and arrays that I had been tripped up on